cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -69,6 +70,7 @@
   private static final Gson GSON = new Gson();
   private static final Type SUMMARY_LIST_TYPE = new TypeToken<List<DatasetSpecificationSummary>>() { }.getType();
   private static final Type MODULE_META_LIST_TYPE = new TypeToken<List<DatasetModuleMeta>>() { }.getType();
+  private static final HttpRequestConfig REQUEST_CONFIG = new HttpRequestConfig(60000, 60000);
```

Comment:
I think if this is above any "reasonable" value, then it need not be configurable. We tested on HDInsights, and 30 was sufficient. So we're making it 60 to be on the safe side.

If someone's network is so much slower than that, I don't think we can expect the overall system to work anyways.
@dereklwood Any input?